documentation/docs/install-pmm/install-pmm-client/connect-database/mongodb.md

```diff
@@ -59,7 +59,6 @@ Run the example codes below in a `mongo` session to:
             pwd: "pmm",
```

Comment:
header a few lines above says `Ceate` instead of `Create`
https://github.com/percona/pmm/pull/3406/files#diff-6fcf3750c550d8a40010282d247ce76dd74eafd646a128ec5c153e440b6e5386R56

Reply:
There is spelling github action in percona-toolkit, maybe we should reuse it